products/ol8/profiles/stig_gui.profile

```diff
@@ -1,10 +1,13 @@
 documentation_complete: true
 
+metadata:
+    version: V1R2
+
```

Comment:
@Xeicker Maybe you want to update the OL8 STIG profile metadata too? Since `stig_gui` extends `stig`.

Reply:
that was addressed in https://github.com/ComplianceAsCode/content/pull/8759